tests/functional/all_daemons/hosts

```diff
@@ -30,7 +30,7 @@ rbd-mirror0
 iscsi-gw0 ceph_repository="dev"
 
 [all:vars]
-nfs_ganesha_stable=True
-nfs_ganesha_dev=False
+nfs_ganesha_stable=False
+nfs_ganesha_dev=True
 nfs_ganesha_stable_branch="V2.7-stable"
```

Comment:
does this still required because we're using ganesha dev ?

Reply:
it gets changed by the dev_setup.yml playbook